patches/tModLoader/Terraria/ModLoader/Logging.cs

```diff
@@ -47,6 +47,8 @@ public enum LogFile
 
 	internal static void Init(LogFile logFile)
 	{
+		LegacyCleanups();
```

Comment:
Maybe move this after the `-build` guard

Reply:
It's in front of the guard because `-build` produced the file being removed there.